cads_catalogue_api_service/client.py

```diff
@@ -362,6 +362,7 @@ def collection_serializer(
     db_model: cads_catalogue.database.Resource,
     request: fastapi.Request,
     preview: bool = False,
+    get_all: bool = True,
```

Comment:
@cettina-tosto why this `get_all` is needed? We already have `preview` for this I think

Reply:
Oh okok